<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OSC_GeneratorDockWidgetBase</class>
 <widget class="QDockWidget" name="OSC_GeneratorDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Vehicles</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QComboBox" name="vehicleSelection">
      <item>
       <property name="text">
        <string>Audi A2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Audi eTron</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Audi TT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BH Crossbike</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BMW Grandtourer</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BMW iSetta</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Carla Cola Truck</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Chevrolet Impala</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Citroen C3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Diamondback Century</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dodge Charger Police</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gazelle Omafiets</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Harley Davidson Low Rider</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Jeep Wrangler</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Kawasaki Ninja</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lincoln MKZ 2017</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mercedes Benz Coupe</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mini Cooper ST</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ford Mustang</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nissan Micra</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nissan Patrol</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Seat Leon</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tesla Cybertruck</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tesla Model 3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Toyota Prius</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Volkswagen T2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Yamaha YZF</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="vehicleOrientation">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>90</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QGroupBox" name="agentGroup">
      <property name="title">
       <string>Agent Selection</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QCheckBox" name="agent_AttachCamera">
         <property name="text">
          <string>Attach Camera</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QComboBox" name="agentSelection">
         <item>
          <property name="text">
           <string>simple_vehicle_control</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>external_control</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="vehicleOrientation_Label">
      <property name="text">
       <string>Orientation</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="vehicleSelection_Label">
      <property name="text">
       <string>Vehicle</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="vehicleIsHero">
      <property name="text">
       <string>Ego Vehicle</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="vehicleOrientation_useLane">
      <property name="text">
       <string>Use lane heading</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="vehicleInitSpeed">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="vehicleInitSpeed_Label">
      <property name="text">
       <string>Init Speed</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="AddVehicleButton">
      <property name="text">
       <string>Insert</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="vehicleLabels">
      <property name="text">
       <string>Labels</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>vehicleSelection</tabstop>
  <tabstop>AddVehicleButton</tabstop>
  <tabstop>vehicleOrientation</tabstop>
  <tabstop>vehicleOrientation_useLane</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
